jupyterlab/packages/jupyterlab-jupytext-extension/ui-tests/README.md

```diff
@@ -0,0 +1,167 @@
+# Integration Testing
+
+This folder contains the integration tests of the extension.
```

Comment:
Very impressive! And helpful as it will allow us to make future changes! Thanks @mahendrapaipuri